flocker/dockerplugin/_api.py

```diff
@@ -231,9 +231,15 @@ def got_configured(configured):
         if profile:
             metadata[PROFILE_METADATA_KEY] = profile
 
+        SIZE = DEFAULT_SIZE
```

Comment:
`ALL_CAPS` is only for constants, so this should probably be `size` instead. Also, consider writing this like
```python
if ...:
   size = ...
else:
   size = ....
```
instead of
```
size =
if ...:
  size =
```
The former makes it clearer that you are picking between two alternatives for what `size` can be.